SmartTutor/Repository/ChallengeProjectRepository/IChallengeProjectRepository.cs

```diff
@@ -0,0 +1,9 @@
+﻿using SmartTutor.ActiveEducationModel;
+
+namespace SmartTutor.Repository.ChallengeProjectRepository
+{
+    public interface IChallengeProjectRepository
```

Comment:
Since we'll be looking at challenges through the LO view, we can merge this with that part of the content model.